Fixing subprocess call for standalone pyinstaller exectuables without console

2press opened the pull request:
To be able to run pytesseract from a standalone executable without a console created by pyinstaller I had to use proc = subprocess.Popen(command, **subprocess_args(True)) with subprocess_args from https://github.com/pyinstaller/pyinstaller/wiki/Recipe-subprocess, whereas e-dul's fix didn't work. Moreover subprocess_args seems to be written in a way that is not platform specific.

int3l commented:
Hi @2press , thank you for your contribution!
I have only one comment - it would be a good idea to strip the comments from the function, right?
We can leave a single comment with the github url link reference, since it's a wiki documentation link.
This way it will be more readable and compact, because right now there is more comments than code within the function.

Do you agree?
And again, thank you very much.

int3l commented on src/pytesseract.py:
```diff
@@ -108,7 +147,7 @@ def run_tesseract(input_filename,
         command.append(extension)
 
-    proc = subprocess.Popen(command, stderr=subprocess.PIPE)
+    proc = subprocess.Popen(command, **subprocess_args(True))
```
Passing the value True, seems to be redundant, since we have default value for the argument (which also defaults to True).

2press commented:
I agree with you in every point. Therefore, I have removed all comments and add a single comment referring to the wiki documentation. Moreover, I removed the redundant argument.
